wazo_confd/plugins/phone_number/notifier.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2024 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from wazo_bus.resources.phone_number.event import (
+    PhoneNumberCreatedEvent,
+    PhoneNumberDeletedEvent,
+    PhoneNumberEditedEvent,
+)
+
+from wazo_confd import bus
+
+from .schema import PhoneNumberSchema
+
+
+class PhoneNumberNotifier:
+    def __init__(self, bus):
+        self.bus = bus
+
+    def created(self, resource):
+        phone_number_serialized = PhoneNumberSchema().dump(resource)
```

Comment:
I usually/often see a singleton instance defined globally for marshmallow schemas, since most schemas are not stateful. This avoids having to create single use instances.